calculator/models.py

```diff
@@ -1,3 +1,78 @@
 from django.db import models
+from django.utils import timezone
 
-# Create your models here.
+
+class Loans(models.Model):
+
+    amount = models.FloatField()
+    term = models.PositiveIntegerField()
+    rate = models.FloatField()
+    date = models.DateTimeField(default=timezone.now)
+    created_loans = 0
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.created_loans += 1
```

Comment:
I think it's best to use django's `post_save`, it's not legal to increment values in class init